app/src/main/java/com/infomaniak/mail/workers/DraftsActionsWorker.kt

```diff
@@ -383,7 +396,13 @@ class DraftsActionsWorker @AssistedInject constructor(
                             Sentry.captureMessage("Return JSON for SendDraft API call was modified")
                         }
                     }
-                    else -> throwErrorAsException()
+                    else -> {
+                        if (isFirstTime && error?.code == ErrorCode.IDENTITY_NOT_FOUND) {
+                            return updateSignaturesThenRetry(draft, mailboxUuid, okHttpClient)
+                        } else {
+                            throwErrorAsException()
+                        }
+                    }
```

Comment:
Would it make sens to factorize that part, maybe something like in [Infomaniak/android-kMail/tree/try-to-factorize](https://github.com/Infomaniak/android-kMail/tree/try-to-factorize)